sr_demos/scripts/sr_demos/demo.py

```diff
@@ -154,345 +172,376 @@ def run(self):
+            Output: demo_states - dictionary of joint states with correct prefix for the robot type
+        '''
+        demo_states = {}
+        for joint_state_dicts_no_id in joint_states_config.keys():
+            joints_target = {}
+            for key, value in joint_states_config[joint_state_dicts_no_id].items():
+                if len(self.prefixes) == 2:
+                    joints_target['rh_' + key] = value
+                    joints_target['lh_' + key] = value
+                else:
+                    joints_target[self.prefixes[0] + key] = value
+                demo_states[joint_state_dicts_no_id] = joints_target
+        return demo_states
+
+    def execute_command_check(self, joint_state_name, time_to_execute,
+                              sleep=0.1,
+                              wait=True, angle_degrees=True):
+        '''
+            Execute a command using the hand commander method move_to_joint_value_target_unsafe
+            input: joint_state_name - name of the joint state to execute
+                    time_to_execute - time to execute the joint state
+                    sleep - time to sleep after executing the joint state
+                    wait - wait for the joint state to finish executing
+                    angle_degrees - if the joint state is in degrees or radians
+        '''
+        if joint_state_name in self.demo_joint_states.keys():
+            if "LFJ" in joint_state_name and self.hand_type != "hand_e":
+                return
+            if "RFJ" in joint_state_name and self.hand_type == "hand_extra_lite":
+                return
+            self.commander.move_to_joint_value_target_unsafe(self.demo_joint_states[joint_state_name],
+                                                             time_to_execute, wait,
+                                                             angle_degrees)
+            rospy.sleep(sleep)
+
+    def check_touched_finger(self):
+        '''
+            Checks if a tactile sensor has been touched
+            and returns the finger that has been touched
+        '''
+        touched_finger = None
+        if len(self.tactiles) == 2:
+            # confirm_touched() will return None if no sensors are found
+            touched_right = robot.tactiles[0].confirm_touched()
+            touched_left = robot.tactiles[1].confirm_touched()
+            if touched_right is not None and touched_left is not None:
+                rospy.loginfo("You touched fingers on both hands at the same time. Defaulting to right touch")
+                touched_finger = touched_right
+            elif touched_right is not None:
+                touched_finger = touched_right
+            elif touched_left is not None:
+                touched_finger = touched_left
+        # check if tactile sensors have been previously found
+        elif len(robot.tactiles) == 1:  # Unimanual mode
+            touched_finger = robot.tactiles[0].confirm_touched()
```

Comment:
```suggestion
            touched_finger = self.tactiles[0].confirm_touched()
```

Reply:
fixed in 92637be0ff846ca31b76aae33b809ec2e931900b